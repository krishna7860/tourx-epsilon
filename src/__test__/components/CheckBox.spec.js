import React from "react";
import { render } from "@testing-library/react";
import StyledCheckbox from "../../components/Checkbox/Checkbox";

describe("components/checkBox", () => {
  it("should be rendered properly", () => {
    const { asFragment } = render(<StyledCheckbox />);
    expect(asFragment()).toMatchSnapshot();
  });
});
