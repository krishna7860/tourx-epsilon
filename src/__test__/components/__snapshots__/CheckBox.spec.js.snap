// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/checkBox should be rendered properly 1`] = `
<DocumentFragment>
  <span
    class="sc-bdnxRM bYuVkg"
    color="#000000"
    height="24px"
    rotate="0"
    width="24px"
  >
    <svg
      class=""
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M416 448H96a32.09 32.09 0 01-32-32V96a32.09 32.09 0 0132-32h320a32.09 32.09 0 0132 32v320a32.09 32.09 0 01-32 32z"
        fill="none"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="32"
      />
    </svg>
  </span>
</DocumentFragment>
`;
